detect_location_weather.py

```diff
@@ -72,6 +71,7 @@ def weather(ip_coordinates, city, state):
     #return("Right now in "+ city + ", " + state + " it is " + temperature +  degree_sign + " and there  is a " + RAIN_WARNING)
     location = {'city': city, 'state': state}
     weather_info = {'temperature' : temperature, 'rain' : rain_commentary}
+    print("Current Weather in"+location(city)+"."+"Temperature is"+weather+info(temperature)+rain_commentary)
```

Comment:
I don't think an extra print statement should be added. Individual developers can add extra prunt statements where they see fit.